src/seedu/addressbook/parser/Parser.java

```diff
@@ -158,6 +159,33 @@ private static boolean isPrivatePrefixPresent(String matchedPrefix) {
         return new HashSet<>(tagStrings);
     }
 
+    /**
+     * Parses arguments in the context of edit person command
+     * @param args String containing the exact name of the person
+     * @return the prepared command
+     */
+    private Command prepareEdit(String args) {
+        StructEdit editParameters = new StructEdit();
+        try {
+            Scanner sc = new Scanner(args).useDelimiter("edit ");
+            sc.useDelimiter(" ");
+
+
+            editParameters.id = Integer.parseInt(sc.next());
+            editParameters.fieldType = sc.next();
+            editParameters.newField = sc.next();
+        } catch(RuntimeException rte) {
+            return new IncorrectCommand(String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditCommand.MESSAGE_USAGE));
+        }
+
+
+//        try{
+//
```

Comment:
It's better to unused code or comments from your PR to improve readability. You can always get the code back from your previous commit in Git history.